client/layout/masterbar/item.tsx

```diff
@@ -139,10 +204,16 @@ class MasterbarItem extends Component< MasterbarItemProps > {
 		}
 
 		return (
-			<button { ...attributes } ref={ this.props.innerRef as LegacyRef< HTMLButtonElement > }>
-				{ this.renderChildren() }
-				{ this.renderSubItems() }
-			</button>
+			<div ref={ this.componentDivRef }>
```

Comment:
I needed to add a wrapper here to apply this ref. The button element below already has a ref passed in from props used for other purposes. Unfortunately, this innerRef from props doesn't always encapsulate the full component items so does not fit our purposes for checking if a touch event started from within the component.

Reply:
Comparing between the two PR's the help icon doesn't load anymore. Could it be this ref change?

Reply:
Which help icon?  Help center? SEems to be there in both my and Dave's testing 🤔

Reply:
Ah I think I see what you mean. The icon is there but it doesn't open the help center in a touch flow. Is that what you were seeing?

It wasn't because of the ref, it was because the `toggleMenuByTouch` function was being added below and not checking for subItems. I overlooked that the below case also renders for items without subItems, so the natural `preventDefault` from that toggle function was preventing the help center from opening. We are checking for subItems now and that should be fixed.

Reply:
Working now, thank you!